CMakeLists.txt

```diff
@@ -196,6 +196,12 @@ if (NOT ANDROID)
         set(JAVA_AWT_LIBRARY NotNeeded)
         set(JAVA_AWT_INCLUDE_PATH NotNeeded)
     endif()
+    if (DEFINED ENV{OVERRIDE_JAVA_HOME})
+        # This override might be set by builder to avoid using the JAVA_HOME modified by gradle
+        file(TO_CMAKE_PATH "$ENV{OVERRIDE_JAVA_HOME}" OVERRIDE_JAVA_HOME)
```

Comment:
I wish we knew better why this was happening, but I'm ok with this as a workaround to get us unstuck